Formula/mongosh.rb

```diff
@@ -17,7 +17,7 @@ class Mongosh < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "f7e6111437d3f1c5aeb2bd9e6189ff2d261e5ae1a2d90126a9cb09c9c6e714c8"
   end
 
-  depends_on "node"
+  depends_on "node@16"
```

Comment:
Can we use the latest LTS, at least?

Reply:
Unfortunately not :( The networking changes that break mongosh are also present in Node.js 18.